core-web/libs/sdk/experiments/src/lib/dot-experiments.ts

```diff
@@ -466,12 +498,19 @@ export class DotExperiments {
 
         if (experimentsData.length > 0) {
             const { experiments } = parseDataForAnalytics(experimentsData, this.currentLocation);
+
             this.analytics.set({ experiments });
+
             this.logger.log('Analytics client updated with experiments data.');
         } else {
             this.analytics.set({ experiments: [] });
             this.logger.log('No experiments data available to update analytics client.');
         }
+
+        // trigger the page view event
+        if (this.config.trackPageView !== false) {
```

Comment:
```suggestion
        if (this.config.trackPageView) {
```

Default this to `false` if is the case.

Reply:
done.